src/main/webapp/js/datepicker.js

```diff
@@ -73,22 +73,6 @@ function triggerDatepickerOnClick(datepickerDivs) {
  * @assumption: startDate has a valid value
  * @returns
```

Comment:
Now that you mention it, this is happening on line 92 as well. Shall I open a first timer issue for this?